_docs-sources/infrastructure-pipelines/security/branch-protection.md

```diff
@@ -0,0 +1,49 @@
+# Branch Protection
+
+:::info Newer Version Available
+This documentation pertains to an old version of Gruntwork Pipelines which used the `infrastructure-pipelines` repository. [Click here](../../pipelines/overview/) to view documentation for the most recent version.
+:::
+
+Gruntwork Pipelines is designed to be used with a PR based workflow.
```

Comment:
```suggestion
Gruntwork Pipelines is designed to be used with a pull-request (PR) based workflow.
```

Some users may not be familiar with that acronym. That being said, I think we only need to define it once per page.